src/iotile/models/datablock.ts

```diff
@@ -17,5 +18,14 @@ export class DataBlock {
     this.sensorGraphSlug = data.sg;
     this.createdOn = new Date(data.created_on);
     this.createdBy = data.created_by;
+    this.description = data.description;
```

Comment:
Both for backwards compatibility and because description may be null, protect with
```
if ('description' in data) {
    this.description = data.description;
}
```